functions/Remove-DbaDbView.ps1

```diff
@@ -0,0 +1,116 @@
+function Remove-DbaDbView {
+    <#
+    .SYNOPSIS
+        Removes a database view(s) from each database and SQL Server instance.
+
+    .DESCRIPTION
+        Removes a database view(s), with supported piping from Get-DbaDbView.
+
+    .PARAMETER SqlInstance
+        The target SQL Server instance or instances. This can be a collection and receive pipeline input to allow the function to be executed against multiple SQL Server instances.
+
+    .PARAMETER SqlCredential
+        Login to the target instance using alternative credentials. Accepts PowerShell credentials (Get-Credential).
+
+        Windows Authentication, SQL Server Authentication, Active Directory - Password, and Active Directory - Integrated are all supported.
+
+        For MFA support, please use Connect-DbaInstance.
+
+    .PARAMETER Database
+        The target database(s).
+
+    .PARAMETER View
+        The name(s) of the view(s).
+
```

Comment:
The Get- command has a parameter ExcludeSystemView - what do you think: Would it be worth adding this here as well?

Reply:
Good idea! However, we can add it in a later PR, let's get this one merged.

Reply:
I agree.